modules/trusted-firmware-m/tfm_boards/common/nrf_provisioning.c

```diff
@@ -16,6 +16,69 @@
 #include "nrf_provisioning.h"
 #include <identity_key.h>
 #include <tfm_spm_log.h>
+#ifdef NRF53_SERIES
+#include <dfu/pcd_common.h>
+#include <spu.h>
+#include <hal/nrf_reset.h>
+
+#define DEBUG_LOCK_TIMEOUT_MS 3000
+#define USEC_IN_MSEC 1000
+#define USEC_IN_SEC 1000000
+
+volatile static struct pcd_cmd *cmd = (struct pcd_cmd *)PCD_CMD_ADDRESS;
+
+static void pcd_write_cmd_lock_debug(void)
```

Comment:
Why are these static functions here and not exposed by the pcd header file itself? Wouldn't it be better isolation if it was handled in the pcd module?

Reply:
A good suggestion. These can be moved to common header file. This just means that pcd_cmd will have to go there as well. This would then affect pcd.c as well. I'll take a look at this.